Apps/Sandcastle/gallery/Image-Based Lighting.html

```diff
@@ -107,15 +107,19 @@
 
     subscribeParameter('luminanceAtZenith');
 
-    Sandcastle.addToggleButton('Use procedural image', false, function(checked) {
-        if (!checked) {
-            model.sphericalHarmonicCoefficients = coefficients;
-            model.specularEnvironmentMaps = environmentMapURL;
-        } else {
-            model.sphericalHarmonicCoefficients = undefined;
-            model.specularEnvironmentMaps = undefined;
-        }
-    });
+    if (Cesium.OctahedralProjectedCubeMap.isSupported(viewer.scene.context)) {
```

Comment:
`OctahedralProjectedCubeMap` is private and so is `scene.context`. You'll need to add a new supported check, probably to `Scene` - `scene.specularEnvironmentMapsSupported`